### PR TITLE
Update MIGRATING.md

### PR DESCRIPTION
**Pull request by arvindkrishnakumar-okta**
## Description
Update migration guide notes on delete group rules API.

## Category

- [ ] Bugfix
- [ ] Enhancement
- [ ] New Feature
- [ ] Configuration Change
- [ ] Versioning Change
- [ ] Unit Test(s)
- [x] Documentation

### CODE REVIEW

File: MIGRATING.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ As part of this upgrade, we will remove it from the method signatures.
    - Removed `getAllGroupsValid` and `setAllGroupsValid` (property `allGroupsValid` is being removed; 
    this option would be removed from future versions of Okta API)
    - Removed `getEmbedded` (property `_embedded` is removed; this property is undocumented in Okta API and hence being removed)
-   - `delete(Boolean removeUsers)` Signature changed to `delete()`
+   - delete(Boolean removeUsers) Signature changed to delete(); optional parameter `removeUsers` which defaulted to `false` earlier is now removed.
```
Comment (arvindkrishnakumar-okta) [inline MIGRATING.md:69]: Callers who used `removeUsers` value `true` will be impacted.

Comment (bdemers) [reply MIGRATING.md:69]: Chat with the team that owns this API, the default may have been changed (or maybe it always removes the users now?)

**Comment by dstiers11** [comment]
Request

File: MIGRATING.md (hunk 1 above)
Comment (arvindkrishnakumar-okta) [reply MIGRATING.md:69]: Sorry this one got buried and I lost track.

So, the user will not removed when the group rule is removed.

**Pull request closed without merging**